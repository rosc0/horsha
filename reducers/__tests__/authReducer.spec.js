import authReducer, { initialState as state } from '../auth'
// import types from '../../constants/actionTypes'

// import { horse_picture, horse_auth } from '../../../test/api'

describe('Auth Reducer', () => {
  it('should return initial state', () => {
    expect(authReducer()).toBe(state)
    expect(authReducer()).toMatchSnapshot()
  })
})
