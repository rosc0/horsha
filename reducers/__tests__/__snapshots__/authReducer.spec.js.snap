// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Auth Reducer should return initial state 1`] = `
Object {
  "auth": Object {
    "accessToken": null,
    "expireDate": null,
    "refreshToken": null,
  },
  "checking": false,
  "error": null,
  "fetched": false,
  "fetching": false,
  "loggedIn": false,
  "resetPassword": Object {
    "errored": false,
    "fetching": false,
    "fulfilled": false,
  },
}
`;
